documentation/src/docs/wire-protocol.md

```diff
@@ -11,102 +11,97 @@ You may obtain a copy of the License at
 
-### Setup Append - Request
+# Appending
 
-1.  ConnectionId - UUID (16 bytes) Identifies this appender.
-2.  Segment to append to. - String (2 byte length, followed by that many bytes of Java's Modified UTF-8)
+## Setup Append - Request
 
-### Append Setup - Reply
+1.  `ConnectionId`: UUID (16 bytes) Identifies this appender.
+2.  Segment to append: String (2 bytes length, followed by that many bytes of Java's Modified UTF-8).
 
-1.  Segment that can be appended to. - String (2 byte length, followed by that many bytes of Java's Modified UTF-8)
-2.  ConnectionId - UUID (16 bytes) Identifies the requesting appender.
-3.  ConnectionOffsetAckLevel - Long (8 bytes) What was the last offset received and stored on this segment for this connectionId (0 if new)
+## Append Setup - Reply
 
-### BeginAppendBlock - Request 
+1.  Segment that can be appended: String (2 bytes length, followed by that many bytes of Java's Modified UTF-8).
+2.  `ConnectionId`: UUID (16 bytes) Identifies the requesting appender.
+3.  `ConnectionOffsetAckLevel`: Long (8 bytes) (The last offset received and stored on this Stream Segment for this `ConnectionId` (0 if new).
 
-Only valid after SetupAppend has already been done successfully.
+## BeginAppendBlock - Request 
 
-1.  ConnectionId - UUID (16 bytes)
-2.  ConnectionOffset - Long (8 bytes) Data written so far over this connection to this segment
-3.  Length of data before EndAppendBlock message - Integer (4 bytes) 
+Only valid after `SetupAppend` has already been done successfully.
 
-### EndAppendBlock- Request
+1.  `ConnectionId`: UUID (16 bytes).
+2.  `ConnectionOffset`: Long (8 bytes) (Data written so far over this connection to this Stream Segment).
+3.  Length of data before `EndAppendBlock` message - Integer (4 bytes). 
 
-1.  ConnectionId - UUID (16 bytes)
-2.  ConnectionOffset - Long (8 bytes) Data written so far over this connection
-3.  Block Length - (4 Bytes) Total size of the block that was written. (Note this may more or less than the number of bytes between the BeginAppendBlock and this message)
+## EndAppendBlock- Request
 
-### Event - Request
+1.  `ConnectionId`: UUID (16 bytes).
+2.  `ConnectionOffset`: Long (8 bytes) Data written so far over this connection.
+3.  Block Length (4 bytes): Total size of the block that was written. (Note that this may be more or less than the number of bytes between the `BeginAppendBlock` and this message).
 
-Only valid inside of a block
+## Event - Request
+
+Only valid inside the block.
 
 1.  Data
 
-### Data Appended - Reply
+## Data Appended - Reply
 
-1.  ConnectionId - UUID (16 bytes)
-2.  ConnectionOffsetAckLevel - Long (8 bytes) The highest offset before which all data is successfully stored on this segment for this connectionId
+1.  `ConnectionId`: UUID (16 bytes).
+2.  `ConnectionOffsetAckLevel`: Long (8 bytes). The highest offset before which all the data is successfully stored on this Stream Segment for this `connectionId`.
```

Comment:
@tkaitchuck @howdy `The highest offset before which all the data is successfully stored on this Stream Segment for this `connectionId`
Highest offset ? need clarity

Reply:
This is out of date. The fields are:
        final UUID writerId;
        final long eventNumber; <- this matches the lastEventNumber in the append block
        final long previousEventNumber;  <- this is the previous value of eventNumber that was returned in the last DataAppeneded

Reply:
Edited! Please check